apps/artifactory/sbox-intsvc/base/kustomization.yaml

```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 resources:
   - ../../base
   - ../../artifactory/sbox-intsvc-ingress.yaml
```

Comment:
is the intention now to keep existing deployment and run this alongside til we know it works?
You'll have to add some DNS records for this new ingress as well if going down this route - and then knock on effects of cleanup after / changing this elsewhere

Worth just replacing?